fastn-core/src/auth/github/mod.rs

```diff
@@ -29,9 +30,22 @@ pub async fn login(
 
     // Note: public_repos user:email all these things are github resources
     // So we have to tell oauth_client who is getting logged in what are we going to access
-    let (authorize_url, _token) = fastn_core::auth::github::utils::github_client(ds)
-        .await
-        .unwrap()
+    let github_client = match fastn_core::auth::github::utils::github_client(ds).await {
+        Ok(client) => client,
+        Err(e) => {
+            // [ERROR] logging (Environment Error)
+            let log_err_message = format!("environment: {:?}", &e);
+            req.log(
+                "github-login",
+                fastn_core::log::OutcomeKind::error_descriptive(log_err_message),
```

Comment:
Here it is not clear what is the outcome. Error message is too never a good thing as it keeps changing, we need all this data for querying not for human consumption.